seacrowd/sea_datasets/id_vaccines_tweets/id_vaccines_tweets.py

```diff
@@ -0,0 +1,124 @@
+&amp;lt;p&amp;gt;&amp;lt;br /&amp;gt;&amp;lt;strong&amp;gt;Kata Kunci:&amp;lt;/strong&amp;gt; covid’19, vaksinasi,
+pelaksanaan vaksin, kartu/sertifikat vaksin, sentimen&amp;lt;br /&amp;gt;analisis&amp;lt;/p&amp;gt;}, number={2},
+journal={Khazanah: Jurnal Mahasiswa}, author={Febriyanti, Syintya and Nursidah, Dea Ratu and Gustiara, Dela and
+Yulianti, Rika}, year={2022}, month={May} }
+"""
+
+_DESCRIPTION = """\
+Dataset containing tweets about COVID-19 vaccines with manually labelled information about whether they are a
+subjective tweet and their sentiment polarity. Tweets are from 20-27 June 2021 and 15-22 July 2021.
+"""
+
+_HOMEPAGE = "	https://github.com/rayendito/id-vaccines-tweets"
+
+_LICENSE = Licenses.UNKNOWN.value
+
+_URL = "https://raw.githubusercontent.com/rayendito/id-vaccines-tweets/main/id_vaccines_tweets.csv"
+
+_SUPPORTED_TASKS = [Tasks.SENTIMENT_ANALYSIS]
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+
+class IdVaccinesTweetsDataset(datasets.GeneratorBasedBuilder):
+    """This is a seacrowd dataloader for id_vaccines_tweets, for every example in the dataset, it contains a subjective
+    tweet and their sentiment polarity.  Tweets are from 20-27 June 2021 and 15-22 July 2021."""
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_source",
+            version=datasets.Version(_SOURCE_VERSION),
+            description=_DESCRIPTION,
+            schema="source",
+            subset_id=f"{_DATASETNAME}",
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_seacrowd_text",
+            version=datasets.Version(_SEACROWD_VERSION),
+            description=_DESCRIPTION,
+            schema="seacrowd_text",
+            subset_id=f"{_DATASETNAME}",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_source"
+
+    def _info(self):
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "idx": datasets.Value("string"),
+                    "form_text": datasets.Value("string"),
+                    "norm_text": datasets.Value("string"),
+                    "subjective": datasets.Value("float"),
+                    "sentiment": datasets.Value("float"),
+                }
+            )
+        elif self.config.schema == "seacrowd_text":
+            features = schemas.text_features([-1.0, 0.0, 1.0])
+        else:
+            raise ValueError(f"Invalid config: {self.config.name}")
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        """ "return splitGenerators"""
+        downloaded_files = dl_manager.download_and_extract(_URL)
+        return [datasets.SplitGenerator(name="datasets.ALL", gen_kwargs={"filepath": downloaded_files})]
```

Comment:
Can you use `datasets.Split.TRAIN` instead of `"datasets.ALL"`? I've checked with the dataset owner (i.e., @rayendito) that all of the data belongs to the train split. I've also fixed the relevant datasheet.